src/pages/api/event/[id]/repairers.ts

```diff
@@ -0,0 +1,27 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import apiHandler from "@/lib/api-handler";
+import { EventRepairer } from "@/types";
+
+import prisma from "../../../../lib/prisma";
+
+export default apiHandler({
+  post: createRepairer
+});
+
+async function createRepairer(
+  req: NextApiRequest,
+  res: NextApiResponse<EventRepairer>
+) {
+  const { userId } = req.body;
+  const { id } = req.query;
+
+  const newEventRepairer = await prisma.eventRepairer.create({
```

Comment:
Should handle the promise rejection instead of always returning 200

Reply:
@dct0 The [apiHandler](https://github.com/codersforcauses/repair-labs/blob/main/src/lib/api-handler.ts) actually has default error handling so it will automatically return 500 for any uncaught error.